src/lib/containers/EntityLink.tsx

```diff
@@ -52,21 +52,19 @@ export const EntityLink = (props: EntityLinkProps) => {
 
     if (link) {
       return (
-        <p className={className}>
-          <a
-            className={className}
-            target="_blank"
-            rel="noopener noreferrer"
-            href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Synapse:${entity.id!}${
-              versionNumber ? `.${versionNumber}` : ''
-            }`}
-          >
-            {showIcon && (
-              <EntityTypeIcon type={type} style={{ marginRight: '6px' }} />
-            )}
-            {entity[displayTextField]}
-          </a>
-        </p>
+        <a
+          className={className}
+          target="_blank"
+          rel="noopener noreferrer"
+          href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Synapse:${entity.id!}${
+            versionNumber ? `.${versionNumber}` : ''
+          }`}
+        >
+          {showIcon && (
+            <EntityTypeIcon type={type} style={{ marginRight: '6px' }} />
+          )}
+          {entity[displayTextField]}
+        </a>
```

Comment:
Remove the <p> rather than adding custom styling to make it inline. Wrapped in <p> where it was important for styling (for example, in SynapseTableCell renderers)